fv3core/stencils/a2b_ord4.py

```diff
@@ -285,15 +286,20 @@ def a2b_interpolation(
 
     with computation(PARALLEL), interval(...):
         qxx = a2 * (qx[0, -2, 0] + qx[0, 1, 0]) + a1 * (qx[0, -1, 0] + qx)
+        qyy = a2 * (qy[-2, 0, 0] + qy[1, 0, 0]) + a1 * (qy[-1, 0, 0] + qy)
+        # TODO use a function with an offset when that works consistently
```

Comment:
Just a suggestion: We might want to start labeling todos with the person who wrote it, so that we can easily tell who to go to with questions about it. For example
```
# TODO(rheag): use a function with an offset when that works consistently
```

Reply:
We can, though it's possible to tell who wrote each line of code in the git history